docs/users-guide/command_line_tools.md

```diff
@@ -69,38 +80,62 @@ Options:
 ```
 
 ### Generate Scala
+
 ```shell
 morphir-elm gen -t Scala
 ```
 
 Options include:
+
+```shell
+  -e, --target-version <version>                                   Scala language version to Generate. (default: "2.11")
+  -c, --copy-deps                                                  Copy the dependencies used by the generated code to the output path. (default: false)
+```
+
+```shell
+morphir scala-gen
+```
+
+Options include:
+
 ```shell
+  -i, --input <path>                                               Source location where the Morphir IR will be loaded from. (default: "morphir-ir.json")
+  -o, --output <path>                                              Target location where the generated code will be saved. (default: "./dist")
   -e, --target-version <version>                                   Scala language version to Generate. (default: "2.11")
   -c, --copy-deps                                                  Copy the dependencies used by the generated code to the output path. (default: false)
+  -m, --limitToModules <comma.separated,list.of,module.names>      Limit the set of modules that will be included. (default: '')
+  -s, --include-codecs <boolean>                                   Generate the scala codecs as well (default: false)
+  --generate-test-generic                                          Generate generic test cases from morphir tests that can be used for testing. (default: false)
+  --generate-test-scalatest                                        Generate runnable scalatest test cases (default: false)
```

Comment:
Maybe we should start adding in the docs when the feature became available. Like in the Java docs: since 2.73.0. That also ensures that we don't forget to release the feature before the docs are updated.